locales/base/translation.json

```diff
@@ -1990,10 +1990,11 @@
     "subtitle": "What would you like to do first?",
     "later": "I'll explore later",
     "options": {
-      "add": "Add funds",
+      "add": "Add funds to your wallet",
```

Comment:
@kathaypacific Should I also create different key for this one where only the wording changed? Or if it doesn't involve functional changes - it can be the same key?

Reply:
yes it can stay in the same key if we're happy to ship this change to all app versions (which i think is the case here)